src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -448,13 +479,17 @@ EscrowFinish::doApply()
             return tecCRYPTOCONDITION_ERROR;
     }
 
-    // NOTE: Escrow payments cannot be used to fund accounts.
     AccountID const destID = (*slep)[sfDestination];
     auto const sled = ctx_.view().peek(keylet::account(destID));
     if (!sled)
         return tecNO_DST;
 
-    if (ctx_.view().rules().enabled(featureDepositAuth))
+    if (ctx_.tx.isFieldPresent(sfCredentialIDs))
+    {
+        if (credentials::removeExpired(view(), ctx_.tx, j_))
+            return tecEXPIRED;
+    }
```

Comment:
Please add a comment here saying no additional CredentialID verification is needed since it's done in preclaim. I was confused for a second about where the doApply logic was.

Reply:
I think there is no need for such a comment as verification is the purpose of preclaim/preflight. Everything in doApply is expected to be verified already.